maintainers/maintainer-list.nix

```diff
@@ -3192,6 +3192,10 @@
     email = "matthew@wellquite.org";
     name = "Matthew Sackman";
   };
+  mschneider = {
+    email = "markus.schneider.sic+nix@gmail.com";
+    name = "Markus Schneider";
```

Comment:
Add yourself as a maintainer in a separate commit in this PR

Reply:
Done